index.js

```diff
@@ -124,13 +124,13 @@ const startAnalysis = async () => {
       const version = issue.platform === 'ios' ? assessmentVersion[0].platform.latestVersion : assessmentVersion[1].platform.latestVersion;
       const singleIssueData = {
         [uuid]: {
-          key: issue.key,
-          title: issue.title,
-          description: issue.description,
-          recommendation: issue.recommendation,
-          severity: issue.severity,
-          assessmentVersion: version,
-          platform: issue.platform
+          key: issue.key || '',
+          title: issue.title || '',
+          description: issue.description || '',
+          recommendation: issue.recommendation || '',
+          severity: issue.severity || '',
+          assessmentVersion: version || '',
+          platform: issue.platform || ''
```

Comment:
In those cases, it might be useful to inform the user that we cannot retrieve anything back from NowSecure in order not to be stunned with the empty result.